CFAAction/CFACustomAction.m

```diff
@@ -9,24 +9,97 @@
 
 #import "CFACustomAction.h"
 
+@interface CFACustomAction()
+    - (void)update;
+    - (void)killDisplayLink;
```

Comment:
Can you remove all spacing before the method prototypes and make it match the other anonymous extensions in the framework?